src/main/java/com/artipie/rpm/pkg/PrimaryOutput.java

```diff
@@ -33,9 +33,8 @@
 /**
  * Package output for {@code primary} metadata.
  * @since 0.4
- * @todo #124:30min Modify this class and XmlPrimary to be able to work with already existing
- *  primary.xml: we need to accept this this xml data about new packages and delete data about
- *  package by package id (checksum). Do fot forget about tests.
+ * @todo #138:30min Make this class be able to work this already existing primary by
```

Comment:
@olenagerasimova don't you need to mention how to use new `XmlPrimaryMaid` in `PrimaryOutput`?

Reply:
@g4s8 no, this todo is basically only for a new ctor, that can accept existing not empty tmp file and test for it. I'll implement usage of this new `XmlPrimaryMaid` in a separate PR.